1001-2000/LIT1664Homily.xml

```diff
@@ -4,11 +4,12 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
    <teiHeader xml:base="https://betamasaheft.eu/">
       <fileDesc>
          <titleStmt>
-            <title xml:lang="gez" xml:id="t1">ድርሳን፡ ዘአንበሮ፡ አብ፡ ቅዱስ፡ ቴዎፍሎስ፡ ሊቀ፡ ጳጳሳት፡ ዘእለስክንድርያ፡ ዘይቤ፡ በእንተ፡ ቤተ፡ ክርስቲያኖሙ፡ ለ፫ደቂቅ፡ ለአናንያ፡
+            <title xml:lang="en" xml:id="t1">Homily on the Church of Three Youths in the Fiery Furnace in Alexandria</title>
+            <title xml:lang="gez" xml:id="t2">ድርሳን፡ ዘአንበሮ፡ አብ፡ ቅዱስ፡ ቴዎፍሎስ፡ ሊቀ፡ ጳጳሳት፡ ዘእለስክንድርያ፡ ዘይቤ፡ በእንተ፡ ቤተ፡ ክርስቲያኖሙ፡ ለ፫ደቂቅ፡ ለአናንያ፡
             ወአዛርያ፡ ወሚሳኤል፡ በዕለተ፡ ተዝካሮሙ፡ ቅዱስ፡ አመ፡ ፲ለግንቦት፡</title>
-            <title xml:lang="gez" type="normalized" corresp="#t1">Dǝrsān za-ʾanbaro ʾab qǝddus Tewoflos liqa ṗāṗṗāsāt za-ʾƎlaskǝndǝryā
```

Comment:
In t1 there's more text, maybe add "za-yǝbe baʾǝnta beta krǝstiyānomu la-3daqiq .... "